README.md

```diff
@@ -32,7 +32,7 @@ gem 'aws-ssm-env', group: :aws
 ```ruby
 # config/application.rb
 if defined?(AwsSsmEnv)
-  AwsSsmEnv.load(path: "/myapp/#{ENV['RAILS_ENV']}", recursive: true)
+  AwsSsmEnv.load(path: "/myapp/#{ENV['RAILS_ENV']}", recursive: true, scope: Settings)
```

Comment:
@dominic-oconnor 
What kind of use case do you assume? What is the motivation to use `Settings`?